.github/DESIGN.md

```diff
@@ -15,6 +26,30 @@ The user understands the basic mechanics of a CLI:
 - CTRL + C quits any running task
 - how to navigate and manipulate the filesystem via the command line: cd and mkdir
 
+## Components of a well designed command
+When creating a new command or adding to an existing command there are a few things to keep in mind.
+
+### Reduce user input
+When trying to create parity with an existing GUI based feature, consider the larger task at had and build your command to accomplish that goal. 
+
+For example when creating a dev-store the data model requires up to 8 fields to be filled in. When creating a CLI command, the ideal experience woulnt be to ask the user for all 8 of those fields - the ideal experience would be to auto-fill those fields with smart generated data. If a field requires an address dont fill it with an animal name, try to get as close as possible to the expected data.
```

Comment:
`If a field requires an address dont fill it with an animal name, try to get as close as possible to the expected data.` 
This feels awkward to me? @gfscott